c7n/resources/account.py

```diff
@@ -367,6 +367,38 @@ def process(self, resources, event=None):
         return []
 
 
+@filters.register('access-analyzer')
+class AccessAnalyzer(ValueFilter):
+    """Check for access analyzers in an account
+
+    :example:
+
+    .. code-block:: yaml
+
+      policies:
+        - name: account-access-analyzer
+          resource: account
+          filters:
+            - type: access-analyzer
+              key: '[].status'
+              value: ACTIVE
+              op: contains
+    """
+
+    schema = type_schema('access-analyzer', rinherit=ValueFilter.schema)
+    schema_alias = False
+    permissions = ('access-analyzer:ListAnalyzers',)
+
+    def process(self, resources, event=None):
+        if not resources[0].get('c7n:access_analyzers'):
+            client = local_session(self.manager.session_factory).client('accessanalyzer')
+            resources[0]['c7n:access_analyzers'] = self.manager.retry(
+                client.list_analyzers)['analyzers']
+        if self.match(resources[0]['c7n:access_analyzers']):
```

Comment:
we should match on any of the access analyzers, ie instead of processing them as a set, we should processing as them individual datums. then if any of the set matches we return the account resource.

Reply:
Makes sense, I think I made that change now.